packages/playground-defaults/src/util.js

```diff
@@ -0,0 +1,49 @@
+const {defaultLookup} = require("./definitions")
```

Comment:
They seem quite generic to me, wouldn't it be better in the core?

Reply:
I would suggest not to do so since:
* It would require installing the whole core just to add default insertion in this case (the main advantage of making `defaults` an own package would be lost)
* I cannot come up with any other use case for this construct I made up from the specification table, but maybe I'm missing something here?

Reply:
Hmm I was thinking that core is always installed but you are right that this can be actually used on its own. It is ok like this :)